.storybook/test-runner.ts

```diff
@@ -0,0 +1,12 @@
+// TODO: Convert this to ESM once the bug is resolved
+// https://github.com/storybookjs/test-runner/issues/293
+
+const config = {
+  async postRender(page, _context) {
+    const elem = await page.$('#story-root');
+    const html = await elem?.innerHTML();
```

Comment:
If elem is null we would be awaiting on null, which probably would cause an error 🤔

Reply:
Ideally it should never be null since we're declaring it as a decorator and it should always exist.

Although awaiting something that's not a Promise, awaiting should return itslef.

```typescript
const x = await null; // x is null
const y = await undefined; // y is undefined
cont z = 23; z is 23
```

The actual problem here would be that if `elem` is undefined, it would resolve `elem.innerHTML` to be undefined and `elem.innerHTML()` to be an error since you can't call a function on an undefined. That can however be fixed using `elem.innerHTML?.()`

Reply:
This probably happens due to how the `await` is internally converted to a Promise, as pretty much it becomes either Promise.resolve(null) or new Promise(resolve => resolve(null);

Yet, I'd rather have we throwing an error if element is null as it pretty much is a precondition for our test-runner.

Reply:
Added an error check